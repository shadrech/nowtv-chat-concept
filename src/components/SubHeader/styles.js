import styled from "react-emotion";

import { MenuItem } from "../Header/styles";

export const SubHeaderWrapper = styled.div`
  height: 3.2rem;
  padding: 0 15rem;
  background: #283765;
  display: flex;
  align-items: center;
  justify-content: space-between;
  position: relative;
`;

export const SubMenuItem = styled(MenuItem)`
  &:after {
    content: "";
    position: absolute;
    bottom: 0;
    left: 0;
    width: 100%;
    height: 0.4rem;
    opacity: ${props => props.active ? 1 : 0};
  }
  &:nth-child(1) {
    &:after {
      background: #e72f8d;
    }
  }
  &:nth-child(2) {
    &:after {
      background: #653796;
    }
  }
  &:nth-child(3) {
    &:after {
      background: #4fade7;
    }
  }
  &:nth-child(4) {
    &:after {
      background: #37ac47;
    }
  }
  &:nth-child(5) {
    &:after {
      background: #5926c0;
    }
  }
  &:nth-child(6) {
    &:after {
      background: #cc091c;
    }
  }

  &:hover {
    background: none;
    &:after {
      opacity: 1;
    }
  }

  p {
    letter-spacing: 0.05rem;
    font-size: 0.9rem;
  }
`;