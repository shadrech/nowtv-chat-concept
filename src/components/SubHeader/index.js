import React, {Component} from "react";

import { SubHeaderWrapper, SubMenuItem } from "./styles";
import { ItemText } from "../Header/styles";
import { Chevron } from "../../assets/icons";

class Header extends Component {
  render() {
    return (
      <SubHeaderWrapper>
        <SubMenuItem>
          <ItemText>About NOW TV</ItemText>
          <Chevron />
        </SubMenuItem>
        <SubMenuItem>
          <ItemText>What's on</ItemText>
        </SubMenuItem>
        <SubMenuItem>
          <ItemText>TV Passes</ItemText>
          <Chevron />
        </SubMenuItem>
        <SubMenuItem active>
          <ItemText>Chat Log</ItemText>
        </SubMenuItem>
        <SubMenuItem>
          <ItemText>NOW TV Stick</ItemText>
        </SubMenuItem>
        <SubMenuItem>
          <ItemText>Get NOW TV</ItemText>
          <Chevron />
        </SubMenuItem>
      </SubHeaderWrapper>
    )
  }
}

export default Header;