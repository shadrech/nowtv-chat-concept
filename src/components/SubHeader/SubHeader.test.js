import React from "react";

import SubHeader from ".";

describe("<SubHeader />", () => {
    test("should match snapshot", () => {
        expect(<SubHeader />).toMatchSnapshot();
    })
})