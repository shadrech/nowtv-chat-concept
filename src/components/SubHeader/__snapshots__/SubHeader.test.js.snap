// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SubHeader /> should match snapshot 1`] = `<Header />`;
